.devcontainer/devcontainer.json

```diff
@@ -3,27 +3,28 @@
 {
 	"name": "C# (.NET)",
 	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
-	"image": "mcr.microsoft.com/devcontainers/dotnet:1-9.0",
+	"image": "mcr.microsoft.com/devcontainers/dotnet:1-9.0-bookworm",
```

Comment:
Why?

Reply:
change docker base image to ubuntu
This is right now the only way I know to interact with windows sshd reliabel (for example if people want to use their local ssh config)